src/pages/userManagement/UserManagement.jsx

```diff
@@ -22,6 +22,7 @@ import usePostUser from '../../models/users/usePostUser';
 import useGetUsers from '../../models/users/useGetUsers';
 import UserEditTable from './UserEditTable';
 import roleSchema from './constants/roleSchema';
+import CollaborationManagementForm from '../collaborations/collaborationManagementForm'; // TODO not sure whether this is where the form component should live. Likely /components instead??
```

Comment:
it's ok where you have it, since it is only used on one page. The `/components` directory is mainly for UI that is repeated in multiple places